acceptance/localcluster/localcluster.go

```diff
@@ -330,6 +338,21 @@ func (l *Cluster) startNode(i int) *Container {
 		"--gossip=" + strings.Join(gossipNodes, ","),
 		"--scan-max-idle-time=200ms", // set low to speed up tests
 	}
+	if (logDirectory != nil) && (len(*logDirectory) > 0) {
```

Comment:
Lots of unnecessary parentheses here: `if logDirectory != nil && len(*logDirectory) > 0`